tests/sn_9_sort_by_alpha.py

```diff
@@ -0,0 +1,73 @@
+import unittest
+from selenium import webdriver
+from selenium.webdriver.chrome.service import Service
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions
+from selenium.webdriver.support.wait import WebDriverWait
+from webdriver_manager.chrome import ChromeDriverManager
+from tests.search_employee import login
+
+
+# functions
+def get_data_from_table(browser, get_column):
+    """selecting conumn and saving value from each row into the list, returns list"""
+    data_list = []
+    column_with_data = browser.find_elements(By.XPATH, get_column)  # locator for name column "//tbody//tr//td[3]"
+
+    for row in column_with_data:
+        data_list.append(row.text)
+
+    return data_list
+
+def get_rest_of_data(browser, list):
+    """finding next page locator, if exists -> gets values, goes to the next, returns extended list"""
+    next_page_button = f"//a[@class = 'current']/following::a[text() = 2]"
```

Comment:
What will happen if there are only 25 records, and there is no pagination at all?